### PR TITLE
Feature/s3 config cloudtrail replication

### PR DESCRIPTION
**Pull request by lgallard**
## What?
* Add support for AWS CloudTrail Logs bucket replication
* Add AWS Config for secondary region

## Why?
* In order to have trails replica dated in the secondary region
* to enable AWS Config on the secondary region

CC: @diego-ojeda-binbash

### CODE REVIEW

**Review by exequielrafaela — APPROVED** [review]

File: apps-devstg/us-east-1/security-audit/awscloudtrail.tf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module "cloudtrail" {
-  source                        = "github.com/binbashar/terraform-aws-cloudtrail.git?ref=0.20.1"
-  namespace                     = var.project
+  source                        = "github.com/binbashar/terraform-aws-cloudtrail.git?ref=0.21.0"
+  iamespace                     = var.project
```
Comment (exequielrafaela) [inline apps-devstg/us-east-1/security-audit/awscloudtrail.tf:3]: @lgallard does this variable changed from namespace to iamspace? Just wondering why this change.

**Review by exequielrafaela — APPROVED** [review]
@lgallard Great job here :) Let's merge this 1st part till we then have the 2nd AWS Config upgrade. Thanks!

File: apps-devstg/us-east-1/security-audit/awscloudtrail.tf (hunk 1 above)
Comment (lgallard) [reply apps-devstg/us-east-1/security-audit/awscloudtrail.tf:3]: This was my bad, I'm going to fix it in next PR @exequielrafaela
